webpack.config.js

```diff
@@ -50,6 +50,7 @@ module.exports = (env) => {
       bundlesLandingPage: 'pages/bundles-landing/bundlesLanding.jsx',
       monthlyContributionsPage: 'pages/monthly-contributions/monthlyContributions.jsx',
       monthlyContributionsThankyouPage: 'pages/monthly-contributions-thankyou/monthlyContributionsThankyou.jsx',
+      monthlyContributionsExistingPage: 'pages/monthly-contributions-existing/monthlyContributionsExisting.jsx',
```

Comment:
Since each of the pages has a `simpleHeader` and a `simpleFooter`, Is it worth to try to implement a [shared bundle strategy](https://webpack.js.org/plugins/commons-chunk-plugin/#components/sidebar/sidebar.jsx)?